FilePersistence/src/version_control/merge/ListMergeComponentV2.h

```diff
@@ -40,6 +41,13 @@ class FILEPERSISTENCE_API ListMergeComponentV2 : public MergePipelineComponent
 		static bool isList(const QString& type);
 		static bool isUnorderedList(const QString& type);
 		static bool isOrderedList(const QString& type);
+
+		QList<Model::NodeIdType> computeListsToMerge(MergeData& mergeData);
+		QMultiHash<Model::NodeIdType, QPair<QString, QString>>
```

Comment:
I just added the `MergeChange::Branches` type, use it in the `QPair` instead of string to indicate which branch makes a change.